### PR TITLE
fix(project/new): extend new route from form route

### PR DESCRIPTION
**Pull request by velrest**
Since we moved the fetchProject call from the {{did-insert}} to the setupController
we need to extend the new route from the edit route.

### CODE REVIEW

File: addon/routes/project/new.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import Route from "@ember/routing/route";
+import FormRoute from "./form";
 import { inject as service } from "@ember/service";
 import ConstructionProject from "ember-ebau-gwr/models/construction-project";
 
-export default class ProjectNewRoute extends Route {
+export default class ProjectNewRoute extends FormRoute {
```
Comment (luytena) [inline addon/routes/project/new.js:5]: From where is the FormRoute?

Comment (velrest) [reply addon/routes/project/new.js:5]: `import FormRoute from "./form";`

Comment (luytena) [reply addon/routes/project/new.js:6]: Ah for ember the exported name doesn't have to be the same? e.g. `ProjectFormRoute`?

**Review by luytena — APPROVED** [review]

**Pull request merged**